packages/apps/graph/src/services/node-service.ts

```diff
@@ -30,84 +32,93 @@ export type ChainFungibleAccountDetails = {
   };
 };
 
-function getClient(chainId: ChainId): IClient {
-  return createClient(
-    `http://${dotenv.NETWORK_HOST}/chainweb/0.0/${dotenv.NETWORK_ID}/chain/${chainId}/pact`,
-  );
-}
-
 export async function getAccountDetails(
   fungibleName: string,
   accountName: string,
   chainId: string,
 ): Promise<ChainFungibleAccountDetails | null> {
-  const commandResult = await getClient(chainId as ChainId).dirtyRead(
-    Pact.builder
-      .execution(
-        Pact.modules[fungibleName as 'fungible-v2'].details(accountName),
-      )
-      .setMeta({
-        chainId: chainId as ChainId,
-      })
-      .setNetworkId(dotenv.NETWORK_ID)
-      .createTransaction(),
-  );
+  let result;
+  try {
+    result = (await details(
+      accountName,
+      dotenv.NETWORK_ID,
+      chainId as ChainId,
+      dotenv.NETWORK_HOST,
+      fungibleName,
+    )) as any;
 
-  if (commandResult.result.status !== 'success') {
-    // If the account does not exist on a chain, we get a row not found error.
-    if (
-      (commandResult.result.error as any).message?.includes(
-        'with-read: row not found',
-      )
-    ) {
+    if (typeof result.balance === 'object') {
```

Comment:
Thats a weird behaviour in the pact api. Do you think I should change all numbers to the same format in the client. Perhaps always returning the stringified version
```
12.1 => "12.1"
{ decimal : "12.123123123" } => "12.123123123"
```
or maybe convert `12.1` to `{ decimal: "12.1" }`

Reply:
All values to string seems like the most logical thing to do indeed.